src/test/java/io/debezium/connector/vitess/VtctldConnection.java

```diff
@@ -26,19 +26,6 @@
 
 /** Use VTCtld to do Vitess admin operations */
 public class VtctldConnection implements AutoCloseable {
```

Comment:
Is this class `VtctldConnection` still used somewhere? If not, can it be removed as well?